test.js

```diff
@@ -0,0 +1,45 @@
+import {serial as test} from 'ava';
```

Comment:
Yup, remove it when `kap-plugin-mock-context` is bumped and we'll get this merged :))).